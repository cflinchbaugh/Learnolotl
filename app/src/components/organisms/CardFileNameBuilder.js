import React from 'react';

import InputField from '../formElements/InputField';

function BuildFileName (props) {
    return (
        <div>
            <h1>Build</h1>
            <p>
                Create new Memorinity cards here!
            </p>
            <br/>
            <p>
                You want to learn more than just the few sample cards that came with the app?  No problem!
                It's easy to make your own Memorinity cards and share them with others.
            </p>
            <br/>
            <p>
                First, what do you want this set of cards to be called?  We'll use it as a filename, so please, just letters and underscores.
            </p>

            <InputField {...props.fileIdData} />
        </div>
    );
}

export default BuildFileName;