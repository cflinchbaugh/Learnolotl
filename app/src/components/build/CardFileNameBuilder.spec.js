import React from 'react';
import {shallow} from 'enzyme';
import CardFileNameBuilder from './CardFileNameBuilder';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('CardFileNameBuilder', () => {
    it('renders and matches snapshot', () => {
        const renderer = new ShallowRenderer();

        const tree = renderer.render(
            <CardFileNameBuilder />
        );

        expect(tree).toMatchSnapshot();
    });

});  