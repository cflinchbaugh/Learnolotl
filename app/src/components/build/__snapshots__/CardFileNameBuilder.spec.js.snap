// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardFileNameBuilder renders and matches snapshot 1`] = `
<styled.div
  className="card-file-name-builder-wrapper clearfix"
>
  <InputField />
  <NavigationArrow
    disabled={true}
    onClickFunction={undefined}
    type="next"
  />
</styled.div>
`;
